frontend/src/pages/LoginPage.jsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from 'react'
+import { Navigate, Link } from 'react-router-dom'
+import axios from 'axios'
+
+export default function LoginPage() {
+  const [email, setEmail] = useState('')
+  const [password, setPassword] = useState('')
+  const [success, setSuccess] = useState(false)
+  
+  async function handleLogin(e) {
+    e.preventDefault()
+    try {
+      await axios.post('http://localhost:2003/login', {
+        email,
+        password,
+      })
+      setSuccess(true)
+    } catch (err) {
+      console.log(err)
+    }
+  }
+
+  if (success) {
+    return<Navigate to='/' />
+  }
+
+  return (
+    <div className='mt-16'>
+      <h1 className='text-center mb-4'>Log In</h1>
+      <form className='w-[80%] max-w-md mx-auto' onSubmit={handleLogin}>
+        <div>
+          <label htmlFor='email'>Email address</label>
```

Comment:
Another thing that might look small but is a huge win for accessibility: having a label ABOVE the input field, not just within the input field. since the text on a form field disappears when you start to type, it can be a frustration for some people if there's not an additional label above the field. So in short, this is a good way to do things!

Reply:
So glad you noticed this! There's definitely more needed to make this form accessible, but I was happy to at least be able to order labels and inputs this way.